Library/Homebrew/cmd/search.rb

```diff
@@ -141,13 +141,20 @@ def search_tap(user, repo, rx)
     []
   end
 
-  def search_formulae(rx)
+  # @param [Regexp] rx
+  # @param [Hash] options
+  # @option options [Boolean] :installed If true, include installed formulae
+  #
+  # @return [Array<String>]
+  def search_formulae(rx, options = { :installed => true })
     aliases = Formula.alias_full_names
```

Comment:
It's better to use an empty hash here and use `Hash#fetch` to deal with defaults. See [cmd/tap.rb](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/tap.rb#L62-L67) for an example.